build-pipelines/buildpacks/pipeline.yml.erb

```diff
@@ -0,0 +1,177 @@
+resources:
+- name: ci
+  type: git
+  source:
+    uri: <%= ci_repo %>
+    branch: <%= ci_branch %>
+- name: build-image-resource
+  type: git
+  source:
+    uri: <%= build_image_resource_repo %>
+    branch: <%= build_image_resource_branch %>
+- name: s3.fissile-linux
+  type: s3
+  source:
+    bucket: <%= fissile_linux_s3_bucket %>
+    private: true
+    regexp: fissile/develop/fissile-(.*)\.tgz
+- name: s3.fissile-stemcell-version
+  type: s3
+  source:
+    bucket: <%= stemcell_versions_s3_bucket %>
+    region_name: <%= stemcell_s3_bucket_region %>
+    access_key_id: <%= s3_access_key %>
+    secret_access_key: <%= s3_secret_key %>
+    versioned_file: <%= stemcell_version_file %>
+- name: kubecf
+  type: git
+  source:
+    uri: <%= kubecf_repo %>
+    branch: <%= kubecf_branch %>
+    paths:
+    - <%= kubecf_ops_set_suse_buildpacks %>
+    private_key: |
+      <%= github_private_key.gsub("\n", "\n      ") %>
+
+<% releases.each do |release| %>
+- name: <%= release[:name] %>-gh-release
+  type: github-release
+  source:
+    owner: SUSE
+    repository: <%= release[:uri].match(/.*\/(.*)\.git/)[1] %>
+    access_token: <%= github_access_token %>
+<% end %>
+
+<% releases.each do |release| %>
+- name: suse-final-release-<%= release[:name] %>
+  type: s3
+  source:
+    bucket: suse-final-releases
+    regexp: <%= release[:name] %>-release-(.*).tgz
+    access_key_id: <%= aws_access_key %>
+    secret_access_key: <%= aws_secret_key %>
+    private_key: |
+      <%= github_private_key.gsub("\n", "\n      ") %>
+<% end %>
+
+jobs:
+<% releases.each do |release| %>
+- name: build-image-<%= release[:name] %>
+  plan:
+  - in_parallel:
+    - get: ci
+    - get: build-image-resource
+    - get: s3.fissile-stemcell-version
+      trigger: true
+    - get: s3.fissile-linux
+      trigger: true
+    - get: <%= release[:name] %>-gh-release
+      trigger: true
+    - get: suse-final-release-<%= release[:name] %>
+    - get: kubecf
+  - do:
+    - task: build
+      privileged: true
+      input_mapping:
+        s3.stemcell-version: s3.fissile-stemcell-version 
+        gh_release: <%= release[:name] %>-gh-release
+      params:
+        STEMCELL_REPOSITORY: <%= stemcell_repository %>
+        STEMCELL_VERSIONED_FILE: <%= stemcell_version_file %>
+        REGISTRY_NAME: <%= registry_name %>
+        REGISTRY_ORG: <%= registry_org %>
+        REGISTRY_USER: <%= registry_user %>
+        REGISTRY_PASS: <%= registry_pass %>
+        RELEASE_NAME: <%= release[:name] %>
+      file: ci/build-pipelines/buildpacks/tasks/build.yml
+
+    - task: create-pr
+      privileged: true
+      input_mapping:
+        suse_final_release: suse-final-release-<%= release[:name] %>
+        gh_release: <%= release[:name] %>-gh-release
+      params:
+        GITHUB_TOKEN: <%= github_access_token %>
+        GITHUB_PRIVATE_KEY: |
+         <%= github_private_key.gsub("\n", "\n         ") %>
+        GIT_MAIL: <%= github_username %>
+        GIT_USER: suse-cf-ci-bot
```

Comment:
This should come from a variable as well.

Reply:
The name for this var in concourse secrets is `rubygems-username` which doesn't seems very descriptive that's why I chose to use the var name directly.